hack/api-reference/template/tools.go

```diff
@@ -0,0 +1,18 @@
+// +build tools
+
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This package imports things required by build scripts, to force `go mod` to see them as dependencies
```

Comment:
Yep, it's only declared as go package so that the directory can be vendored by others, see https://github.com/gardener/gardener-extension-provider-aws/blob/f9b9f455c6af7b6e915f3f2de4a438cb4f663bdf/pkg/apis/aws/v1alpha1/doc.go#L20

Reply:
Ah, forgot that the extensions also use these templates...